src/main/java/seedu/taassist/ui/MainWindow.java

```diff
@@ -112,15 +100,22 @@ private void setWindowDefaultSize(GuiSettings guiSettings) {
     }
 
     /**
-     * Opens the help window or focuses on it if it's already opened.
+     * Opens a particular webpage.
+     */
+    public static void openWebpage(String urlString) {
+        try {
+            Desktop.getDesktop().browse(new URL(urlString).toURI());
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+    }
```

Comment:
Perhaps other than printing stack trace, we can also provide a message to the user mentioning that we are unable to redirect them to the website, and provide them the URL directly? Also, we might need to smoke test this on different platforms to ensure that it works properly.

Reply:
ok i guess we can keep the help window for this purpose! But I don't know how it will be triggered, considering that i'm still able to open the webpage offline